### ISSUE TITLE
No more Emacs? 😄 (EOM)

### ISSUE THREAD

**Issue by prometheansacrifice** [sriramkswamy/vscode-dotfiles#1]

**Comment by sriramkswamy** [comment]
Haha, no I still do all the time but that config has been pretty much stable for a while now. I'm tl giving the whole LSP thing a serious go to see how good it is. Emacs doesn't yet seem to have great support for it.

**Comment by prometheansacrifice** [comment]
Right. Good choice since VSCode is supposed to have the best LSP support (no surprises there). Just curious, which lang servers do you need? `lsp-mode` seems to work fine for me with ocaml and clang.

**Comment by sriramkswamy** [comment]
Python (pyls) mostly with some clang and R. Clang and R are very well supported by Emacs already. Python seems to be really slow with lsp-mode though and the maintainers acknowledge this. Unfortunately, it's the one I use the most
